financial-connections/src/main/java/com/stripe/android/financialconnections/presentation/FinancialConnectionsSheetNativeViewModel.kt

```diff
@@ -366,8 +367,8 @@ internal class FinancialConnectionsSheetNativeViewModel @Inject constructor(
             bankAccountToken != null
 
     fun onPaneLaunched(pane: Pane, referrer: Pane?) {
-        // Do not track pane loaded for exit pane as it is not a real pane.
-        if (pane != EXIT) {
+        // Do not track theses panes as they are not "real" panes.
+        if (pane != EXIT && pane != NOTICE) {
```

Comment:
these can be probably a boolean on the Destination object. I'll add a task and work on it on a separate PR